index.html

```diff
@@ -1,12 +1,17 @@
 <html>
   <head>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/4.5.1/pixi.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/5.0.3/pixi.js"></script>
     <script src="dat.gui.min.js"></script>
     <script src="pixi-ae.js"></script>
   </head>
   <body>
     <script>
-      var renderer = new PIXI.WebGLRenderer(2000, 600, { antialias: true, backgroundColor: 0xffffff });
+      var renderer = new PIXI.Renderer({
```

Comment:
Replace the new Class Renderer , drop width and height in the options parameters [ Renderer Parameter ](https://github.com/pixijs/pixi.js/wiki/v5-Migration-Guide)